api/models.py

```diff
@@ -270,6 +275,7 @@ class SituationReport(models.Model):
     document_url = models.URLField(blank=True)
 
     event = models.ForeignKey(Event, on_delete=models.CASCADE)
+    type = models.ForeignKey(DocumentType, related_name='situation_reports', null=True, on_delete=models.CASCADE)
```

Comment:
@szabozoltan69 since we're allowing `null=True`, probably easier for the user to just have `on_delete=models.SET_NULL` so that removing a document type doesn't wipe out all it's associated documents.